GitHub-Test-4/python_excercise1.py

```diff
@@ -0,0 +1,11 @@
+import spacy
+nlp = spacy.load('en_core_web_sm')
```

Comment:
You may want to update to the medium model if you're playing with the cosine similarity!